ui/lib/kubernetes/addon/components/page/role/create-and-edit.hbs

```diff
@@ -116,10 +116,14 @@
             @valueUpdated={{fn (mut template.rules)}}
             @helpText={{sanitized-html this.roleRulesHelpText}}
           >
-            <button type="button" class="toolbar-link" {{on "click" this.resetRoleRules}} data-test-restore-example>
-              Restore example
-              <Icon @name="reload" />
```

Comment:
Based on the original position of the icon, the `@iconPosition` arg should be set to `trailing` but it looks like this goes against [the guidelines](https://helios.hashicorp.design/components/button#leading) so I ignored it. Considering that most icons should be leading I didn't bother writing anything into the codemod to try and discern the position so it will need to be manually audited on a case by case basis.